README.md

```diff
@@ -16,37 +19,59 @@
+    - [Extending pooler with a Uniswap v2 data point](#extending-pooler-with-a-uniswap-v2-data-point)
+      - [Step 1. Review: Base snapshot extraction logic for trade information](#step-1-review-base-snapshot-extraction-logic-for-trade-information)
+      - [Step 2. Review: 24 hour aggregate of trade volume snapshots over a single pair contract](#step-2-review-24-hour-aggregate-of-trade-volume-snapshots-over-a-single-pair-contract)
+      - [Step 3. New Datapoint: 2 hours aggregate of only swap events](#step-3-new-datapoint-2-hours-aggregate-of-only-swap-events)
+  - [2. Zkevm Quests: A Case Study of Implementation](#2-zkevm-quests-a-case-study-of-implementation)
+    - [Review: Base snapshots](#review-base-snapshots)
+    - [`zkevm:bungee_bridge`](#zkevmbungee_bridge)
 - [Find us](#find-us)
 
 ## Overview
 
 ![Snapshotter workflow](snapshotter/static/docs/assets/OverallArchitecture.png)
 
-A snapshotter peer as part of Powerloom Protocol does exactly what the name suggests:  It synchronizes with other snapshotter peers over a smart contract running on the present version of the PowerLoom Protocol testnet. It follows an architecture that is driven by state transitions which makes it easy to understand and modify.
+A snapshotter peer as part of Powerloom Protocol does exactly what the name suggests:  It synchronizes with other snapshotter peers over a smart contract running on Powerloom Prost chain. It follows an architecture that is driven by state transitions which makes it easy to understand and modify.
 
 Because of its decentralized nature, the snapshotter specification and its implementations share some powerful features that can adapt to your specific information requirements on blockchain applications:
 
-* each data point is calculated, updated, and synchronized with other snapshotter peers participating in the network
+* Each data point is calculated, updated, and synchronized with other snapshotter peers participating in the network
 * synchronization of data points is defined as a function of an epoch ID(identifier) where epoch refers to an equally spaced collection of blocks on the data source blockchain (for eg, Ethereum Mainnet/Polygon Mainnet/Polygon Testnet -- Mumbai). This simplifies the building of use cases that are stateful (i.e. can be accessed according to their state at a given height of the data source chain), synchronized, and depend on reliable data. For example,
     * dashboards by offering higher-order aggregate datapoints
     * trading strategies and bots
 * a snapshotter peer can load past epochs, indexes, and aggregates from a decentralized state and have access to a rich history of data
     * all the datasets are decentralized on IPFS/Filecoin
     * the power of these decentralized storage networks can be leveraged fully by applying the [principle of composability](#aggregation-and-data-composition---snapshot-generation-of-higher-order-datapoints-on-base-snapshots)
 
+### Architecture
```

Comment:
* We should link to the `snapshotter-computes` and `snapshotter-config` repos. 
* Either in this section, or in the case studies section, highlight the computes and configs specific to their snapshotter implementation